Warcraft.NET/Files/ADT/Entries/Legion/MBNVEntry.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.IO;
+using System.Numerics;
+using Warcraft.NET.Extensions;
+
+namespace Warcraft.NET.Files.ADT.Entries.Legion
+{
+    /// <summary>
+    /// Blend mesh vertices entry.
+    /// </summary>
+    public class MBNVEntry
+    {
+        /// <summary>
+        /// Vertex position.
+        /// </summary>
+        public Vector3 Position { get; set; }
+
+        /// <summary>
+        /// Vertex normal.
+        /// </summary>
+        public Vector3 Normal { get; set; }
+
+        /// <summary>
+        /// Texture/UV coordinate.
+        /// </summary>
+        public Vector2 TextureCoordinates { get; set; }
+
+        /// <summary>
+        /// Color.
+        /// </summary>
+        public uint[] Color { get; set; }
```

Comment:
The wiki says its 3 RGBA so maybe you can use here  Warcraft.NET.Files.Structures.RGBA and ReadRGBA/WriteRGBA